product/live_metrics/middleware_server.yaml

```diff
@@ -0,0 +1,24 @@
+#
+# LiveMetrics configuration file
+#
+#   It defines the live metrics supported in a model that includes a LiveMetricsMixin.
+#   It maps the native id used in the provider for a metric with the column where this metric will be available in
+#   a LiveMetric virtual model.
+#
+#   Format:
+#   - Provider native metric id: MiQ metric id
+#
+- WildFly Memory Metrics~Heap Used: mw_heap_used
```

Comment:
The left side of the metric is the identificer of the metric in the native system.
I'm afraid this long name is produced from the agent and it is propagated into Hawkular Inventory.
Here is the point where is translated into a more "friendly" column name to be used inside MiQ.